vignettes/cross-validation.Rmd

```diff
@@ -0,0 +1,101 @@
+---
+title: "Cross-Validation in sgdnet"
+author: "Johan Larsson"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Cross-Validation in sgdnet}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Model validation is essential to both assess the performance of a model
+and to be able to tune its parameters. **sgdnet** fits regularized models that
```

Comment:
I agree with this, but as implemented our CV error may be a biased predictor of "true" test error (esp if we use lambda_min or lambda_1se for model selection)